client/src/pages/OurEvents/OurEvents.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import imgEvent from "~/assets/images/events-hero.jpg";
+
+import "./OurEvents.css";
+import Data from "./OurEvents.json";
+import { FaAngleRight } from "react-icons/fa";
+
+const images = require.context("../../assets/images", true);
+
+function OurEvents() {
+	return (
+		<div className="ourevent_container_main">
+			<div className="ourevent_img">
+				<img src={imgEvent} alt="Event" />
+			</div>
+			<div className="container_cart">
+				{Data.map((event, i) => {
+					const text = document.createElement("div");
```

Comment:
I think data is not a good variable name, you should use something more descriptive, for example events